source/elements/oneTBB/source/constrained_arena_helpers.rst

```diff
@@ -0,0 +1,38 @@
+.. SPDX-FileCopyrightText: 2025 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+=========================
+Constrained Arena Helpers
+=========================
+**[constrained_arena_helpers]**
+
+Helper interfaces to create and manage multiple ``task_arena`` objects with constraints.
+
+.. code:: cpp
+
+    // Defined in header <oneapi/tbb/task_arena.h>
+
+    namespace oneapi {
+    namespace tbb {
+        std::vector<oneapi::tbb::task_arena> create_numa_task_arenas(
+            oneapi::tbb::task_arena::constraints other_constraints = {},
+            unsigned reserved_slots = 0);
+    } // namespace tbb
+    } // namespace oneapi
+
+Functions
+---------
+
+.. cpp:function:: std::vector<oneapi::tbb::task_arena> create_numa_task_arenas(oneapi::tbb::task_arena::constraints other_constraints = {}, unsigned reserved_slots = 0)
+
```

Comment:
As far as I can tell, the specification does not use `oneapi::tbb` in the signatures of functions that are already in that namespace.

Reply:
Removed `oneapi::tbb`